tensorboard/webapp/widgets/line_chart_v2/lib/coordinator_test.ts

```diff
@@ -159,7 +159,7 @@ describe('line_chart_v2/lib/coordinator test', () => {
     });
 
     describe('#transformDataToUiCoord', () => {
-      beforeEach(() => {
+      it('converts with y-axis pointing up', () => {
```

Comment:
Maybe I'm confusing myself, but this test uses the default coordinator with getYAxisDirection -1.  Should the text say "converts with y-axis pointing down"?

Reply:
This is for the ThreeCoordinator where y-axis is pointing up (L142 reads: `describe('ThreeCoordinator', () => {`)

Reply:
Ah, gotcha.  Silly me for not scrolling up to read around this line!